tools/topology/sof-bdw-rt5640.m4

```diff
@@ -97,3 +97,8 @@ DAI_CONFIG(SSP, 0, 0, Codec,
 		      SSP_CLOCK(fsync, 48000, codec_slave),
 		      SSP_TDM(2, 25, 3, 3),
 		      SSP_CONFIG_DATA(SSP, 0, 24)))
+VIRTUAL_WIDGET(codec_out0, output, 1)
+VIRTUAL_WIDGET(codec_out1, output, 2)
+VIRTUAL_WIDGET(codec_in0, input, 3)
+VIRTUAL_WIDGET(codec_in1, input, 4)
```

Comment:
this is not needed, bdw is not impacted - different driver.

Reply:
I've removed them.